spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -311,27 +311,27 @@ class SpdxExpressionTest : WordSpec() {
 
             "correctly convert ORs on both sides of an AND expression" {
                 val spdxExpression = "(a OR b) AND (c OR d)".parse()
-                val dnf = "((a AND c) OR (a AND d)) OR ((b AND c) OR (b AND d))".parse()
+                val dnf = "(a AND c) OR (a AND d) OR (b AND c) OR (b AND d)".parse()
```

Comment:
Can't `isValidChoice()` now also be simplified to basically just check `choice in validChoices()` (in a new commit)?

Reply:
Done.